src/app/modules/layout/components/admin-layout/admin-layout.component.ts

```diff
@@ -38,7 +39,7 @@ export class AdminLayoutComponent implements OnInit, AfterViewInit, OnDestroy {
   readonly isAlertPanelOpen$ = this.store$.select(selectIsAlertPanelOpen);
   readonly hasConsoleFooter$ = this.store$.select(selectHasConsoleFooter);
   readonly productType$ = this.sysGeneralService.getProductType$;
-  readonly copyrightYear$ = this.sysGeneralService.getCopyrightYear$;
+  readonly copyrightYear = environment.buildYear;
```

Comment:
There is a `copyright-line` component that relies on `selectBuildTime`. It needs to be updated.

Reply:
@denysbutenko That component is already fixed. It uses `environment.buildYear` to decide build year.